README.md

```diff
@@ -60,14 +60,22 @@ npm i --save-dev enzyme
 Enzyme is currently compatible with `React 15.x`, `React 0.14.x` and `React 0.13.x`. In order to
 achieve this compatibility, some dependencies cannot be explicitly listed in our `package.json`.
 
-If you are using `React 0.14` or `React 15.x`, in addition to `enzyme`, you will have to ensure that
+If you are using `React 0.14` or `React <15.5`, in addition to `enzyme`, you will have to ensure that
 you also have the following npm modules installed if they were not already:
 
 ```bash
 npm i --save-dev react-addons-test-utils
 npm i --save-dev react-dom
 ```
 
+If you are using `React >=15.5`, in addition to `enzyme`, you will have to ensure that you also have
+the following npm modules installed if they were not already:
+
+```bash
+npm i --save-dev react-dom
+npm i --save-dev prop-types
```

Comment:
Does `prop-types` work with every version of React?

If it does, then let's just depend on it explicitly instead of making it funky. `airbnb-prop-types` will likely depend on it very soon anyways, and enzyme will be depending on that soon if it doesn't already :-)